plugins/ffmpeg_plugin/kahawai.c

```diff
@@ -659,12 +681,18 @@ static const AVOption kahawai_options[] = {
      DEC},
     {"ext_frames_mode",
      "Enable external frames mode",
-     OFFSET(ext_frames),
+     OFFSET(ext_frames_mode),
      AV_OPT_TYPE_BOOL,
      {.i64 = 1},
      0,
      1,
      DEC},
+    {"dma_node",
```

Comment:
nit: usually we use dma_dev for the naming. Can you help to change then we can has same name.

Reply:
Sure, no problem.